source/Ship.cpp

```diff
@@ -2212,6 +2212,10 @@ void Ship::DoGeneration()
 	energy -= ionization;
 	fuel -= leakage;
 	heat += burning;
+
+	recentShield *= 0.75;
+	recentHeat *= 0.984;
```

Comment:
Those numbers seem to pop out of nowhere... They should be constant in the nameless namespace at the start of the file. Also I'm wondering why we bother to calculate recent heat at all; shoudnt we just show how heated the ship is currently?

Also really not a fan of doing these calculations when the preference is disabled, please check for that.

Reply:
It might be more costly to check the preferences every time for one operation?